src/Request.ts

```diff
@@ -178,3 +178,46 @@ export function isRetryable (status?: number): boolean {
       ].indexOf(status) !== -1)
     )
 }
+
+export function fetch(endpoint: string): Promise<string> {
```

Comment:
I'm not sure if this should retry or not, so there's some logic here copied from `send` that deals with that which could be removed, or the request retying from `request` could be incorporated here too

On the one hand the RN bundler will run on localhost a lot of the time, so retrying won't usually make a difference but on the other hand it can be run remotely, where network issues may break things

Reply:
I don't think retry is important here – the `--fetch` variant isn't designed to run automatically as part of a build, but to be invoked manually when the app is running in development. So in that case the user should be able to see the error and try again.